client/shipping-label/views/purchase/steps/packages/index.js

```diff
@@ -1,56 +1,129 @@
 import React, { PropTypes } from 'react';
 import { translate as __ } from 'lib/mixins/i18n';
-import OrderPackages from './list';
+import PackageList from './list';
+import PackageInfo from './package-info';
+import MoveItemDialog from './move-item';
+import Unpacked from './unpacked';
 import FormButton from 'components/forms/form-button';
 import { hasNonEmptyLeaves } from 'lib/utils/tree';
 import { sprintf } from 'sprintf-js';
 import StepContainer from '../../step-container';
 
-const PackagesStep = ( { values, storeOptions, labelActions, errors, expanded } ) => {
-	const packageIds = Object.keys( values );
-	const firstPackageId = packageIds[ 0 ];
-	const firstPackage = values[ firstPackageId ];
-	const isValid = 0 < firstPackage.weight;
+const PackagesStep = ( { openedPackageId, selected, all, unpacked, storeOptions, labelActions, errors, expanded, showItemMoveDialog, movedItemIndex, targetPackageId } ) => {
+	const packageIds = Object.keys( selected );
+	const itemsCount = packageIds.reduce( ( result, pId ) => ( result + selected[ pId ].items.length ), 0 );
+	const totalWeight = packageIds.reduce( ( result, pId ) => ( result + selected[ pId ].weight ), 0 );
```

Comment:
Consider using lodash's [`sumBy()`](https://lodash.com/docs/4.16.4#sumBy) for this.